bitnami/postgresql-ha/Chart.lock

```diff
@@ -3,4 +3,4 @@ dependencies:
   repository: https://charts.bitnami.com/bitnami
   version: 1.10.1
 digest: sha256:84f150f2d532eb5cb38ad0201fc071d7a1c43d1e815330cd8dedd5bc268575ec
-generated: "2021-11-15T17:06:43.180136754Z"
+generated: "2021-11-15T17:06:43.180136754Z"
```

Comment:
Please, keep the file as the original one. I don't have permissions to edit, so I can not fix it. Thanks

Reply:
I'm a bit confused here. Should I revert this back to v1.10.0? Because the change you see now is pulled from the master

Reply:
well you are including now a change. You are removing a newline in the end of the file. You don't need to include any change in this file at all. This file will be change automatically by the pipeline.